docs/_quarto.yml

```diff
@@ -9,66 +9,55 @@ website:
   navbar:
     logo: https://user-images.githubusercontent.com/4471859/224825908-bee7e044-bc6b-4561-8b08-5d330cce3ed5.png
     left:
-      - text: "About"
-        file: index.qmd
-      - text: "Get Started"
-        file: tutorials/index.qmd
-      - text: "Guides"
-        file: guides/index.qmd
-      - text: "Background"
-        file: background/concept.qmd
+      - text: "Tutorials"
+        file: tutorial/index.qmd
+      - text: "How-to guides"
+        file: guide/index.qmd
+      - text: "Explanation"
+        file: explanation/index.qmd
```

Comment:
I don't mind changing any of the titles, but note I consciously chose different names than how diataxis names/splits its system (which is for writers of documentation, not end-users). Indeed, this is fine per [diataxis](https://diataxis.fr/explanation/#explanation-and-understanding):

> Explanation by any other name:
>
>	Your explanation documentation doesn’t need to be called Explanation.
>
>	Alternatives include:
>		
>	- Discussion 
>	- Background
>	- Conceptual guides
>	- Topics

In the different names, I tried to find names that (I think) are clear to end-users, and took inspiration from a page like https://quarto.org/. Maybe @gijsber can comment on the preferred names (feel free to suggest others)?

`Tutorials` or `Get Started`
`Guides` or `How-to guides`
`Background` or `Explanation`

This PR removes also the About header, which is the same as the logo, but for me it's not clear the logo is a link to the home page. I prefer to add it back (can be another name, such as Home).

Reply:
My suggestions would be
`Getting Started`
`How-to`
`Background`
`Reference`

Reply:
I'm fine with adding back the home page, though perhaps "Overview" like in https://quarto.org/ is more accurate and less internet-lingo, especially if we want to make PDF docs out of this as well.

I should've included my reasoning for renaming in the PR description, but I didn't have time anymore. They are:

- Tutorials: Diátaxis says they are for learning. This may include advanced topics in tutorial format. For that "Getting started" does not seem the right title. Quarto.org uses this and then combines a How-to install guide with some tutorials. Installation is a more of a guide, not a tutorial.
- How-to guides: They are guides that tell you how to do something. I think it's clearer to write it out, it's not long.
- Explanation: I feel like "Background" is much narrower that "Explanation". Some of the content there is background information, but not all that is needed for understanding is background information. Also background seems less important.

Reply:
ok. Fine with your explanation to go for `Tutorials`, `How-to guides`.
Although I am fine with `Explanation`, we might also consider if `Understanding` is an appropriate alternative. Other words, like `Concepts` can be too narrow and `Background` may indeed not trigger people to read and understand.

Reply:
Understanding is the goal. But we cannot do that, only provide a good explanation and hope it leads to understanding. I think that is why it is not in the alternative names list that @evetion linked to.